R/pool.R

```diff
@@ -19,24 +18,33 @@ NULL
 #' @export
 Pool <- R6Class("Pool",
   public = list(
+
     valid = NULL,
     counters = NULL,
     minSize = NULL,
     maxSize = NULL,
     idleTimeout = NULL,
+    validateTimeout = NULL,
```

Comment:
Yes, I agree. Also, thanks for the post :) It did seem like things started taking exponentially more time to get done... I was commenting with Winston yesterday, that it's like that 80/20 rule: first, you do 80% of the work in 20% of the time. And you feel really good about that. But then, the last 20% of the work take up 80% of the time...